vault/ha.go

```diff
@@ -245,9 +245,9 @@ func (c *Core) StepDown(httpCtx context.Context, req *logical.Request) (retErr e
 	if te != nil && te.NumUses == tokenRevocationPending {
 		// Token needs to be revoked. We do this immediately here because
 		// we won't have a token store after sealing.
-		leaseID, err := c.expiration.CreateOrFetchRevocationLeaseByToken(te)
+		leaseID, err := c.expiration.CreateOrFetchRevocationLeaseByToken(ctx, te)
```

Comment:
Might as well use the active context here too since if this fails the below wont be run